typescript/infra/src/govern/HyperlaneAppGovernor.ts

```diff
@@ -112,14 +116,33 @@ export abstract class HyperlaneAppGovernor<
       }
 
       console.log(
-        `> ${callsForSubmissionType.length} calls will be submitted via ${SubmissionType[submissionType]}`,
+        `${SubmissionType[submissionType]} calls: ${callsForSubmissionType.length}`,
+      );
+      callsForSubmissionType.map(
+        ({ icaTargetChain, description, expandedDescription, ...call }) => {
+          // Print a blank line to separate calls
+          console.log('');
```

Comment:
considered it, but would complicate the logic because we start this group of logs either inside the if-statement or just after it, depending on whether it's an ICA call